solidity/ecdsa/contracts/WalletRegistry.sol

```diff
@@ -78,15 +84,28 @@ contract WalletRegistry is
     uint256 internal _dkgResultSubmissionGas;
 
     /// @notice Gas that is meant to balance the DKG result approval's overall
-    ///         cost. It can be updated by the governace based on the current
+    ///         cost. It can be updated by the governance based on the current
     ///         market conditions.
     uint256 internal _dkgResultApprovalGasOffset;
 
     /// @notice Gas that is meant to balance the notification of an operator
-    ///         inactivity. It can be updated by the governace based on the
+    ///         inactivity. It can be updated by the governance based on the
     ///         current market conditions.
     uint256 internal _notifyOperatorInactivityGasOffset;
 
+    /// @notice Gas that is meant to balance the notification of a seed for DKG
+    ///         delivery timeout. It can be updated by the governance based on the
+    ///         current market conditions.
+    uint256 internal _notifySeedTimeoutGasOffset;
+
+    /// @notice Gas that is meant to balance the notification of a DKG protocol
+    ///         execution timeout. It can be updated by the governance based on the
+    ///         current market conditions.
+    /// @dev The value is subtracted for the refundable gas calculation, as the
+    ///      DKG timeout notification transaction recovers some gas when cleaning
+    ///      up the storage.
+    uint256 internal _notifyDkgTimeoutNegativeGasOffset;
```

Comment:
It could - theoretically - happen that after some hardfork, and opcode pricing changes, we will need a positive offset so it could make sense to use a signed integer. On the other hand, this is:
1. unlikely to happen
2. DKG timeout is not called often - hopefully never
3. we can always upgrade this contract

Reply:
That's a great catch! I also like that in the same message you provided reasoning for not changing the code. Lazy me appreciates that!